cvat-core/src/object-state.js

```diff
@@ -20,7 +20,9 @@
             * @param {Object} serialized - is an dictionary which contains
             * initial information about an ObjectState;
             * Necessary fields: objectType, shapeType
+            * (don't have setters)
             * Necessary fields for objects which haven't been added to collection yet: frame
+            * (doesn't have setters)
```

Comment:
23 and 25 lines are the same?

Reply:
@nmanovic 

Except of don't, doesn't, yes. 
The main idea is to say that it's need to setup these fields in constructor because you can't change it after (they are readonly)

Two fields are needed always.
And one fields is needed only if it is a new object which haven't been saved before.